### PR TITLE
Update connect-to-remote-aadj-pc.md

### PR DESCRIPTION
**Pull request by RavennMSFT**
added note about RDP from AAD registered device that was enabled in 2004 version

### CODE REVIEW

**Comment by RavennMSFT** [comment]
@Dansimp Need this approved, been pending for a while now

**Comment by RavennMSFT** [comment]
#sign-off

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ From its release, Windows 10 has supported remote connections to PCs joined to A
 ## Set up
 
 - Both PCs (local and remote) must be running Windows 10, version 1607 or later. Remote connections to an Azure AD-joined PC running earlier versions of Windows 10 are not supported.
-- Your local PC (where you are connecting from) must be either Azure AD joined or Hybrid Azure AD joined. Remote connections to an Azure AD joined PC from an unjoined device or a non-Windows 10 device are not supported.
+- Your local PC (where you are connecting from) must be either Azure AD joine or Hybrid Azure AD joined, if using WIndows 10 versio 1607 and above or Azure AD registered, if using Windows 10 version 2004 and above. Remote connections to an Azure AD joined PC from an unjoined device or a non-Windows 10 device are not supported.
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:35]:
```suggestion
- Your local PC (where you are connecting from) must be either Azure AD joined or Hybrid Azure AD joined if using Windows 10 version 1607 and above, or Azure AD registered if using Windows 10 version 2004 and above. Remote connections to an Azure AD joined PC from an unjoined device or a non-Windows 10 device are not supported. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by RavennMSFT** [comment]
@JohanFreelancer9 done

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**